### PR TITLE
Add admin and it's sub routes with navigation and add UI for index route

### LINKED ISSUE #153: Add user admin page  at /admin (UI only)
**I'm submitting a ...**  (check one with "x")
- [ ] bug report
- [x] feature request
- [ ] support request => Please do not submit support requests here, instead ask your query in out 
- Located at `/admin`

image

- eleven tabs (hence eleven routes). This issue is for index route only. Please open other issues for other routes.
    - Admin - `/admin`
    - events - `/admin/events/`
    - sales - `/admin/sales`
    - sessions - `/admin/sessions`
    - Users - `/admin/users/`
    - permissions - `/admin/permissions`
    - messages - `/admin/messages`
    - Reports - `/admin/reports/`
    - settings - `/admin/settings`
    - Modules - `/admin/modules`
    - content - `/admin/contents`

While working on additional issues for the sub-routes of admin, please don't make the entire tab structure for navigating these routes, it will be taken care of it in this issue itself.

### PR DESCRIPTION
**Pull request by CosmicCoder96**
#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

#### Short description of what this resolves:

Adds admin and it's sub routes with navigation and UI for /admin at `admin/index.hbs`
Since a lot of files are changed, I have split this into two commits.

#### Changes proposed in this pull request:
image

Fixes #153

### CODE REVIEW

**Comment by CosmicCoder96** [comment]
@geekyd @niranjan94 Please review

**Comment by geekyd** [comment]
@CosmicCoder96 Can you please provide deployment link?

**Comment by CosmicCoder96** [comment]
@geekyd The deployment link : https://cc96-opev-frontend.herokuapp.com/
Please review.

File: app/templates/admin.hbs [hunk 1]
```diff
@@ -0,0 +1,42 @@
+<div class="row">
+  <div class="sixteen wide column">
+    <div class="ui fluid pointing secondary menu">
+      {{#link-to 'admin.index' class='item'}}
```
Comment (geekyd) [inline app/templates/admin.hbs]: Please wrap this around tabbed-navigation.

**Comment by CosmicCoder96** [comment]
@geekyd I have added tabbed navigation, please review again, thank you
https://cc96-opev-frontend.herokuapp.com/admin

**Pull request merged**